scripts/switch.py

```diff
@@ -944,16 +936,12 @@ def __init__(self,
             raise ValueError(f'Cutout type {cutout} not supported.')
 
         self.mx_cutout = cutout
-        self.hotswap_plated = hotswap_plated
 
         _name = 'SW_Hotswap_Hybrid'
         _tags = 'Kailh Keyboard Keyswitch Switch Hybrid Hotswap Socket'
-        _description = 'Kailh keyswitch Hybrid Hotswap Socket'
+        _description = 'Kailh keyswitch Hybrid Hotswap Socket, keycap generation assumes cherry sizes'
```

Comment:
don't love this for description, not sure I would even add that kind of note as in the future I plan to make this more dynamic